### DISCUSSION TITLE
"I will create an AI that will teach 1000 new AI

### DISCUSSION THREAD

**Discussion by MariuszKedziora** [community/community#101797] [New to GitHub] [answered]
I never learned programming anywhere. I am Polish and 39 years old. Recently, I got fascinated with AI, and in just a few days, I taught myself. A friend who is a programmer told me that if I understand logic, I can handle programming because it's sometimes tricky but generally cool. And indeed, as this little bot starts doing something based on what I wrote, I'm having a great time!

**Comment by bencheikh-la** [comment]
c

**Reply by MariuszKedziora** [reply]
Właśnie nie wiem, bo sprawdzam tę stronę/. Nawet nie wiem o co tu chodzi, ale od wczoraj sie bawie i fajne to :D ps: aha ja po angielsu tam pisalem, na 100%

**Comment by samus-aran** [comment, marked as answer]
Welcome to the GitHub Community, @MariuszKedziora , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by bencheikh-la** [reply]
moi je cherche seulement à utiliser facilement GitHub. merci
